shell/src/main/java/alluxio/shell/command/HeadCommand.java

```diff
@@ -56,10 +58,9 @@ protected int getNumOfArgs() {
   @Override
   protected void runCommand(AlluxioURI path, CommandLine cl) throws AlluxioException, IOException {
     URIStatus status = mFileSystem.getStatus(path);
-
     int numOfBytes = Constants.KB;
     if (cl.hasOption('c')) {
-      numOfBytes = Integer.parseInt(cl.getOptionValue('c'));
+      numOfBytes = (int)parseSpaceSize(cl.getOptionValue('c'));
```

Comment:
/home/jenkins/workspace/Alluxio-Pull-Request-Builder/shell/src/main/java/alluxio/shell/command/HeadCommand.java:33: error: Import statement for 'alluxio.util.FormatUtils.parseSpaceSize' is in the wrong order. Should be in the 'STATIC' group, expecting not assigned imports on this line.
/home/jenkins/workspace/Alluxio-Pull-Request-Builder/shell/src/main/java/alluxio/shell/command/HeadCommand.java:63:25: error: 'typecast' is not followed by whitespace.
/home/jenkins/workspace/Alluxio-Pull-Request-Builder/shell/src/main/java/alluxio/shell/command/TailCommand.java:33: error: Import statement for 'alluxio.util.FormatUtils.parseSpaceSize' is in the wrong order. Should be in the 'STATIC' group, expecting not assigned imports on this line.
/home/jenkins/workspace/Alluxio-Pull-Request-Builder/shell/src/main/java/alluxio/shell/command/TailCommand.java:58:25: error: 'typecast' is not followed by whitespace.